doc/GeoLift_Walkthrough.html

```diff
@@ -340,11 +340,11 @@ <h1 class="title toc-ignore">GeoLift_Walkthrough</h1>
 <h1><code>GeoLift</code>: And end-to-end solution to Measure Lift at a Geo Level</h1>
 <div id="installation" class="section level2">
 <h2>Installation</h2>
-<p>You can install <code>GeoLift</code> from GitHub using the <code>devtools</code> package.</p>
-<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="co">## Install devtools if noy already installed</span></span>
-<span id="cb1-2"><a href="#cb1-2"></a><span class="kw">install.packages</span>(<span class="st">&quot;devtools&quot;</span>, <span class="dt">repos=</span><span class="st">&#39;http://cran.us.r-project.org&#39;</span>)</span>
+<p>You can install <code>GeoLift</code> from GitHub using the <code>remotes</code> package.</p>
+<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="co">## Install remotes if noy already installed</span></span>
+<span id="cb1-2"><a href="#cb1-2"></a><span class="kw">install.packages</span>(<span class="st">&quot;remotes&quot;</span>, <span class="dt">repos=</span><span class="st">&#39;http://cran.us.r-project.org&#39;</span>)</span>
 <span id="cb1-3"><a href="#cb1-3"></a><span class="co">## Install GeoLift from github</span></span>
-<span id="cb1-4"><a href="#cb1-4"></a>devtools<span class="op">::</span><span class="kw">install_github</span>(<span class="st">&quot;ArturoEsquerra/GeoLift&quot;</span>)</span></code></pre></div>
+<span id="cb1-4"><a href="#cb1-4"></a>remotes<span class="op">::</span><span class="kw">install_github</span>(<span class="st">&quot;ArturoEsquerra/GeoLift&quot;</span>)</span></code></pre></div>
```

Comment:
@laresbernardo this should probably also change to `facebookincubator/GeoLift` right? Check aat the end of the line.